qlty-cloud/src/export/coverage.rs

```diff
@@ -70,18 +70,23 @@ impl CoverageExport {
             .cloned()
             .collect();
 
-        compress_files(raw_file_paths, &directory.join("raw_files.zip"))
+        compress_files(raw_file_paths, &directory.join("raw_files.zip"))?;
+
+        let files_to_zip = vec![
+            "report_files.json.gz",
+            "file_coverages.json.gz",
+            "metadata.json",
+            "raw_files.zip",
```

Comment:
What do you think of including raw files directly instead of in a nested zip?

Reply:
Considered (and can still do). The raw files retain the folder structure from the client, so I'd probably want to move them to a new top level folder.